registration-processor/registration-processor-core/src/main/java/io/mosip/registration/processor/core/packet/dto/abis/DataShareResponseDto.java

```diff
@@ -0,0 +1,21 @@
+package io.mosip.registration.processor.core.packet.dto.abis;
+
+import io.mosip.registration.processor.core.common.rest.dto.BaseRestResponseDTO;
+import io.mosip.registration.processor.core.common.rest.dto.ErrorDTO;
+import lombok.Data;
+import lombok.EqualsAndHashCode;
+
+import java.util.List;
+
+@Data
+@EqualsAndHashCode(callSuper = true)
+public class DataShareResponseDto extends BaseRestResponseDTO {

```

Comment:
where are we using this class ?

Reply:
We are using this class while creating data share url in abis handler stage